addons/event/models/event.py

```diff
@@ -402,3 +405,14 @@ def action_send_badge_email(self):
             'target': 'new',
             'context': ctx,
         }
+
+
+class MailComposeMessage(models.Model):
```

Comment:
mail.compose is a TransientModel model. can you correct it ?

and also move this stuff into new python file : mail_compose_message in wizard folder.